src/components/Calculator.js

```diff
@@ -1,25 +1,34 @@
+/* eslint-disable react/state-in-constructor */
 /* eslint-disable react/prefer-stateless-function */
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useState } from 'react';
 import '../index.css';
 import Buttons from './Buttons';
 import Output from './Output';
+import calculate from '../logic/calculate';
 
-// eslint-disable react/prefer-stateless-function
-export default class Calculator extends React.Component {
-  render() {
-    return (
-      <div className="calculator-grid">
-        <Output
-          total={this.props.total}
-          operation={this.props.operation}
-          next={this.props.next}
-        />
-        <Buttons
-          clickHandler={this.props.clickHandler}
-        />
-      </div>
-    );
-  }
+export default function Calculator() {
```

Comment:
- [ ]  Let's update this function to ES6 syntax by using arrow function.

Reply:
Hello @arslanbisharat. Isn't that code from my previous commits out of this pull request?